README.md

```diff
@@ -2,4 +2,30 @@
 
 This repository holds the masterlist prelude, a metadata file that is used to supply common metadata to all masterlists.
 
-See [CONTRIBUTING.md](CONTRIBUTING.md) for information on how to contribute to the prelude.
\ No newline at end of file
+See [CONTRIBUTING.md](CONTRIBUTING.md) for information on how to contribute to the prelude.
+
+### Synchronising Weblate translations
+
+The `translations` directory holds files that are read and written by [Weblate](https://hosted.weblate.org/projects/loot/prelude/). There are a couple of scripts that can be used to keep them in sync with `prelude.yaml`.
+
+To use the scripts, first install their dependencies in a virtual environment. On Windows, make sure Python 3 is installed, then run:
+
+```
+py -m venv .venv
+.\.venv\Scripts\activate
+pip install -r requirements.txt
+```
+
+To regenerate the files in the `translations` directory from `prelude.yaml`, run:
+
+```
+py scripts/export-translations.py
+```
+
+To regenerate the files in the `translations` directory from the `prelude.yaml`, run:
```

Comment:
This sentence is very similar to the one above, so it might be better to write something like:
To update `prelude.yaml` with the contents of the `translations` directory, run:

Reply:
That was due to a bad rebase, I've replaced that line.